re-adding STAR

fhartley101 opened the pull request:
re-addition of star modules
I also change the name of genecounts to summarise_featurecounts to better describe the process

This also fixes the bug I discovered where featureCounts misnamed some files

fhartley101 committed:
re-adding STAR

alebarberis commented on main.nf:
```diff
@@ -111,7 +120,7 @@ def initialLogMessage() {
 
     log.info "\nCUTADAPT ------------------------------"
-    if (params.cutadapt_options) log.info "cutadapt_options = ${params.cutadapt_options}"
+    //is there a way to add the args here
```
If you define the args in the params object you can do like for the others. I am not entirely sure if we can get the information when we define the args in the modules.config file.

fhartley101 replied:
I checked and it doesn't appear to be possible. I print the arguments used in the cutadapt process using log.info so that will have to be sufficient

alebarberis commented on subworkflows/check_and_prepare_input.nf:
```diff
@@ -361,17 +360,35 @@ workflow CHECK_AND_PREPARE_INPUT {
+                            salmon_index
+                        )
+                        // Update
+                        salmon_index = GUNZIP.out.decompressed //possible issue here - should this be ch_salmon_index, or do we want it to go through the next steps?
+                    }
```
I think I already commented on this, salmon_index is correct. We are checking it in the following lines. Please, remove the comment.

fhartley101 committed:
Update check_and_prepare_input.nf

Had deleted this from the github version but not my own - re-fixed

fhartley101 committed:
Update main.nf

removed section for displaying cutadapt options as it doesn't seem to be possible
